Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -32,6 +32,10 @@ Containment:
   same underlying vertex type.
 * `SimpleGraph.Free` is the predicate that `H` is `G`-free, that is, `H` does not contain a copy of
   `G`. This is the negation of `SimpleGraph.IsContained` implemented for convenience.
+* `SimpleGraph.killCopies G H`: Subgraph of `G` that does not contain `H`. Obtained by arbitrarily
+  removing an edge from each copy of `H` in `G`.
+* `SimpleGraph.copyCount G H`: Number of copies of `H` in `G`.
+* `SimpleGraph.labelledCopyCount G H`: Number of labelled copies of `H` in `G`.
```

Comment:
I'd like if we could add a bit more informal description of what this means here.